src/js/fsa/visual_fsa.js

```diff
@@ -276,6 +276,33 @@ export default class VisualFSA extends EventHandler {
         this.dispatchEvent('change')
     }
 
+    editTransition (from, to) {
+        this.removeTransitions(from, to)
```

Comment:
I will look into these changes tonight! Thanks for the review!

Reply:
I renamed the method editTransition to addTransitionOverlay because I feel like it better describes what it does, ie getting the user input symbol with the overlay and hten adding the transition.